src/posthog-featureflags.ts

```diff
@@ -20,17 +26,19 @@ export const parseFeatureFlagDecideResponse = (response: Partial<DecideResponse>
                     $enabled_feature_flags,
                 })
         } else {
-            // using the v2 api
+            // using the v2^ api
```

Comment:
No payloads in v2 right?

```suggestion
            // using the v3 api
```

Reply:
Ah well, combining the two, I see the intention..
```suggestion
            // using the v2+ api
```